src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -26,10 +26,10 @@
 
 [assembly: LoadableClass(typeof(void), typeof(KMeansPlusPlusTrainer), null, typeof(SignatureEntryPointModule), "KMeans")]
 
-namespace Microsoft.ML.Trainers.KMeans
+namespace Microsoft.ML.KMeansClustering
 {
     /// <include file='./doc.xml' path='doc/members/member[@name="KMeans++"]/*' />
-    public class KMeansPlusPlusTrainer : TrainerEstimatorBase<ClusteringPredictionTransformer<KMeansPredictor>, KMeansPredictor>
+    public class KMeansPlusPlusTrainer : TrainerEstimatorBase<ClusteringPredictionTransformer<KMeansModelParameters>, KMeansModelParameters>
```

Comment:
Can Visual Studio retrieve the doc in xml? If not, we need to add enough information here.